Tasks/TwineAuthenticateV0/task.json

```diff
@@ -5,7 +5,7 @@
     "description": "Authenticate for uploading Python distributions using twine. Add '-r FeedName/EndpointName --config-file $(PYPIRC_PATH)' to your twine upload command. For feeds present in this organization, use the feed name as the repository (-r). Otherwise, use the endpoint name defined in the service connection.",
     "author": "Microsoft Corporation",
     "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/package/twine-authenticate",
-    "helpMarkDown": "[More Information](https://pypi.org/project/twine/)",
+    "helpMarkDown": "[Learn more about this task](https://pypi.org/project/twine/) or [see the Twine documentation](https://twine.readthedocs.io/)",
```

Comment:
Should this "Learn more about this task" lead to our own documentation rather than pypi.org?

https://docs.microsoft.com/azure/devops/pipelines/tasks/package/twine-authenticate

Reply:
🤦‍♂ yes